runtime/src/accounts_db.rs

```diff
@@ -831,8 +849,17 @@ impl AccountsDB {
         LittleEndian::write_u64(&mut buf[..], slot);
         hasher.hash(&buf);
 
+        LittleEndian::write_u64(&mut buf[..], rent_epoch);
+        hasher.hash(&buf);
+
         hasher.hash(&data);
 
+        if executable {
```

Comment:
Note to myself: when sanitizing `executable`, check the upper 7-bit are cleared.

Reply:
Reply to myself: yeah, I've done this correctly at #7464